### ISSUE TITLE
Window heading doesn't show

### ISSUE THREAD

**Issue by huseyinemanet** [marekhrabe/messenger#7]
Hi there,
I'm using 10.9.5 and window headings shows up at first launch. But then, it suddenly disappears.

![screen shot 2015-04-13 at 17 43 22](https://cloud.githubusercontent.com/assets/878928/7117930/a507f3a6-e204-11e4-9cff-731a0bc273fc.png)

**Comment by marekhrabe** [comment]
Are there any particular steps that would help me to reproduce this state? I couldn't reproduce it at all and I'm using the app all the time from the release.

**Comment by huseyinemanet** [comment]
Nope. It was smooth when the first version released. I'm just opening it and it looks like this in first sight:

![screen shot 2015-04-15 at 13 04 54](https://cloud.githubusercontent.com/assets/878928/7156543/1957500a-e370-11e4-80e6-c7771a72e736.png)

When the app loads successfully window heading disappears:

![screen shot 2015-04-15 at 13 04 59](https://cloud.githubusercontent.com/assets/878928/7156614/8dc80880-e370-11e4-9a03-2299e995623f.png)
